src/Inputs/Slider.tsx

```diff
@@ -59,6 +62,7 @@ export const Slider: React.FunctionComponent<SliderProps> = ({
     popupwidth = 'auto',
     popupheight = 20,
     theme,
+    maxAndMinDifference = max - min,
```

Comment:
Doesnt this only change when min or max changes? if so place in useeffect and watch the two variables

Reply:
The thing is it never changes because min and max never change... The only reason Im using it is to make the codes shorter and more readable ... I can delete it and replace it everywhere with max-min simply

Reply:
@sahandfarahani Yes do that then